open_spiel/python/mfg/examples/mfg_Proximal_policy_optimization_pytorch.py

```diff
@@ -0,0 +1,448 @@
+import os
+# 
+os.environ["OMP_NUM_THREADS"] = "4" # export OMP_NUM_THREADS=4
+os.environ["OPENBLAS_NUM_THREADS"] = "4" # export OPENBLAS_NUM_THREADS=4 
+os.environ["MKL_NUM_THREADS"] = "4" # export MKL_NUM_THREADS=6 Mainly controlles the number of spawned threateds 
+os.environ["VECLIB_MAXIMUM_THREADS"] = "4" # export VECLIB_MAXIMUM_THREADS=4
+os.environ["NUMEXPR_NUM_THREADS"] = "4" # export NUMEXPR_NUM_THREADS=6
+
+import argparse
```

Comment:
use absl flags. See other examples in OpenSpiel

Reply:
Hello, parameter parser was changed to absl flags